sphinx/source/docs/first_steps/examples/first_steps_4_axes_logarithmic.py

```diff
@@ -0,0 +1,33 @@
+from bokeh.plotting import figure, output_file, show
+
+# prepare some data
+x = [0.1, 0.5, 1.0, 1.5, 2.0, 2.5, 3.0]
+y0 = [i ** 2 for i in x]
+y1 = [10 ** i for i in x]
+y2 = [10 ** (i ** 2) for i in x]
```

Comment:
I think that using "math" spacing between operators, i.e. `alpha*x**k + offset` would be superior to this, especially in the context where (or by whom) this documentation will be consumed.

Reply:
Thanks, this will also be part of my next commit!